tests/test_base.py

```diff
@@ -38,6 +38,7 @@ def test_should_get_credits_balance(self, requests_mock):
 @dataclasses.dataclass(eq=True)
 class ActiveRecord:
     session = base.Session()
+    class_workspace_id = base.WorkspaceId()
```

Comment:
I get it. We should change the workspace_id of the class only, while the global workspace_id shock remain the account workspace_id

Reply:
It's rather two scenarios merged into one.
1. In most of the cases (e.g. asset) a user deals with objects using different workspace ids
2. webhook is a special case, since a user always uses the same workspace id for all operations.

Reply:
At least this is how it was for the current approach in SDK